File: SecondYear/Geek-shreya_ShreyaGarg_2226cse1095_2/Week_1/Pattern_7.py
#User function Template for python3

class Solution:
    def printDiamond(self, N):
        for i in range(1,N+1,1):
            for j in range(0,N-i,1):
                print(" ",end="")
            for k in range(0,i,1):
                print("* ",end="")
            print()
        for i in range(N,0,-1):
            for j in range(0,N-i,1):
                print(" ",end="")
            for k in range(0,i,1):
                print("* ",end="")
            print()

        


#{ 
 # Driver Code Starts
#Initial Template for Python 3

if __name__ == '__main__':
    t = int(input())
    for _ in range(t):
        N = int(input())
        ob = Solution()
        ob.printDiamond(N)
# } Driver Code Ends

File: SecondYear/chaudharyaryanpanwar_AryanPanwar_2226cs1053_2/Week_1/Pattern7.cpp
class Solution {
    public :
    void printDiamond(int n){
        //code here
         for (int i = 1 ;i <= n ; i++ ){
            for (int j = i ; j<n ; j++){
                cout<<" ";
            }
            for (int j = 1; j<=i ; j++){
                cout<<"* ";
            }
            cout<< endl;
            
        }
        for (int i = n ;i >= 1 ; i--){
            for (int j = i ; j<n ; j++){
                cout<<" ";
            }
            for (int j = 1; j<=i ; j++){
                cout<<"* ";
            }
            cout<< endl;
            
        }
    }
};